### ISSUE TITLE
(Menu) Netplay Stateless Mode doesn't save

### ISSUE THREAD

**Issue by klepp0906** [libretro/RetroArch#9425]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

So tonight I have been making a stab at getting netplay up and running.  I refrained previously due to assuming upnp was a requirement, whereas it looks like some manual port forwarding and/or the relay are ways around that.

anyhow, i made it down to the Netplay Stateless Mode setting.   Figured id try it since im on symmetrical gigabit fiber.   Turns out, if you toggle it on, it doesnt save.  The only way i could get it to stick, was through the cfg.  Just the same, if you try to toggle it off at that point, its stuck on until you use the cfg file and change it back to false.

### Expected behavior

The toggle to save in the on or off position upon exit, like all the rest :)

### Actual behavior

The toggle is returning to its original position after a restart of retroarch, whether thats on or off, depends on what you manually set in the cfg ;p

### Steps to reproduce the bug

1. Turn on Netplay Stateless Mode (the default is off)
2. Exit retroarch.
3. Restart RA (or watch the cfg entry in a reloading text editor) and notice it doesnt save your change. 

### Bisect Results

No idea, havent touched netplay a single time until tonight :(

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.78 97d91a895c

### Environment information

- OS: Windows 10

**Comment by RobLoach** [comment]
We unfortunately had to bring down the netplay lobby due to spam. Hopefully we can bring it back up soon.

**Comment by klepp0906** [comment]
yep, I had heard from someone in the discord the lobby was down - fortunately I found out before I ended up banging my head against a wall.

people can be so scummy.   You guys bust your asses to provide something a plethora of people enjoy, and then you get people that do that kinda crap.  NBD though, ive went without netplay thus far, I imagine I can make it a little longer.  The port i forwarded is coming up open so when the servers are back up I ought to be okay anyhow.
